src/style.css

```
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+body {
+  margin-top: 20px;
+  font-family: "Avenir", sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.input {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  margin-bottom: 20px;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+}
+
+.input::placeholder {
+  position: relative;
+  left: 0;
+  right: 20px;
+  top: 0;
+  bottom: -16px;
+  font-family: "Avenir", sans-serif;
+  font-style: normal;
+  font-size: 16px;
```

remove default values
```suggestion
  font-family: inherit;
```